pkg/endpoint/endpoint.go

```diff
@@ -2074,8 +2080,8 @@ func (e *Endpoint) SetStateLocked(toState, reason string) bool {
 		e.getLogger().WithFields(logrus.Fields{
 			logfields.EndpointState + ".from": fromState,
 			logfields.EndpointState + ".to":   toState,
-			"file": fileName,
-			"line": fileLine,
+			"file":                            fileName,
```

Comment:
I had `go fmt` do this in go 1.11, but the CI has a different version of `go fmt`, which causes the formatting precheck to fail. You may want to run `gofmt` of a specific version until we update the CI :/

Reply:
Never mind, the build already got past that point. I stand corrected!